src/execution/sema/sema_builtin.cpp

```diff
@@ -1491,13 +1492,28 @@ void Sema::CheckMathTrigCall(ast::CallExpr *call, ast::Builtin builtin) {
       }
       break;
     }
+    case ast::Builtin::Abs: {
+      if (!CheckArgCount(call, 1)) {
+        return;
+      }
+      if (!call_args[0]->GetType()->IsArithmetic()) {
+        // TODO(jkosh44): would be nice to be able to provide multiple types
```

Comment:
I do have some idea of how to accomplish this, it would involve some changes to `Error Reporter`. However I didn't want to include it in this PR because it would be a fairly involved change on it's own and this PR is already big enough.